tests/providers/test_nebius_provider.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from unittest.mock import patch, MagicMock
+
+from aisuite.providers.nebius_provider import NebiusProvider
+
+def test_nebius_provider():
+    """High-level test that the provider is initialized and chat completions are requested successfully."""
+
+    user_greeting = "We are testing you. Please say 'One two three' and nothing more."
+    message_history = [{"role": "user", "content": user_greeting}]
+    selected_model = "Qwen/Qwen2.5-32B-Instruct-fast"
+    chosen_top_p = 0.01
+    response_text_content = "One two three"
+
+    provider = NebiusProvider()
+    print(provider.api_key)
+    response = provider.chat_completions_create(model=selected_model, messages=message_history, top_p=chosen_top_p)
```

Comment:
@Aktsvigun We have been mocking the interaction with the providers. If you look at an example like this one (https://github.com/andrewyng/aisuite/blob/main/tests/providers/test_sambanova_provider.py) we are mostly testing the setup more so than the actual result.

Can you update this to follow that pattern?

Reply:
Sure, fixed! Kindly check. 
I've standardized the test according to the example you sent & modified the NebiusProvider to match most of other providers (e.g. Mistral, Sambanova, etc.)